### PR TITLE
tests: add configtest case for non-existent config

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

test

**Did you add tests for your changes?**
yes

**If relevant, did you update the documentation?**
NA

**Summary**
- Adds a case for non-existent config for configtest command
- 
**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

**Review by jamesgeorge007 — COMMENTED** [review]
Already present.

https://github.com/webpack/webpack-cli/blob/50bbe56c0ae9d72301c4ac51fdc2b04df7b66451/test/configtest/configtest.test.js#L40-L46

**Comment by anshumanv** [comment]
> Already present.

Ah yes missed it, nvm improved existing test to assert on filename

File: test/configtest/configtest.test.js [hunk 1]
```diff
@@ -41,7 +42,7 @@ describe('basic info usage', () => {
         const { exitCode, stderr, stdout } = run(__dirname, ['configtest', './a.js'], false);
 
         expect(exitCode).toBe(2);
-        expect(stderr).toContain(`The specified config file doesn't exist`);
+        expect(stderr).toContain(`The specified config file doesn't exist in '${path.join(__dirname, './a.js')}`);
```
Comment (rishabh3112) [inline test/configtest/configtest.test.js:45]: This change doesn't add any value to test IMO.
The aim for the test is to look for error when configuration doesn't exists, which previous one was also doing. Unnecessary regressions must be avoided as they lead to unstability.

**Review by rishabh3112 — CHANGES_REQUESTED** [review]
Left a comment.

**Comment by anshumanv** [comment]
Yes this seems unnecessary, lets close

**Pull request closed without merging**